README.md

```diff
@@ -7,42 +7,41 @@ Xamarin.Forms provides a way to quickly build native apps for iOS, Android, Wind
 
 Read more about the platform at https://www.xamarin.com/forms.
 
-## Build Status 
+## Build Status ##
 
-![Visual Studio Team Services ](https://img.shields.io/vso/build/devdiv/0bdbc590-a062-4c3f-b0f6-9383f67865ee/7981.svg?style=flat&label=VSTS%20Master "VSTS")
- 
+![Visual Studio Team Services](https://img.shields.io/vso/build/devdiv/0bdbc590-a062-4c3f-b0f6-9383f67865ee/7981.svg?style=flat&label=VSTS%20Master "VSTS")
```

Comment:
Probably should replace this with Azure DevOps. @rmarinho would this work or does the label on the SVG need to change on the build system as well?
```suggestion
![Azure DevOps](https://img.shields.io/vso/build/devdiv/0bdbc590-a062-4c3f-b0f6-9383f67865ee/7981.svg?style=flat&label=Azure%DevOps%20Master "Azure DevOps")
```

Reply:
https://img.shields.io/vso/build/devdiv/0bdbc590-a062-4c3f-b0f6-9383f67865ee/7981.svg?style=flat&label=AzureDevOps%20Master

Reply:
the url is wrong o the label part.